doc/api/fs.md

```diff
@@ -2680,6 +2658,33 @@ The `encoding` option sets the encoding for the `path` while opening the
 directory and subsequent read operations (unless otherwise overriden during
 reads from the directory).
 
+## fs.openSync(path[, flags, mode])
```

Comment:
This ordering is wrong. The sync versions should directly follow the async versions or else this doc will become horribly confusing.

Reply:
If I understand correctly, we never made exceptions in ABC order. See in the current doc:
```
fs.read(fd, buffer, offset, length, position, callback)
fs.readdir(path[, options], callback)
fs.readdirSync(path[, options])
fs.readFile(path[, options], callback)
fs.readFileSync(path[, options])
fs.readlink(path[, options], callback)
fs.readlinkSync(path[, options])
fs.readSync(fd, buffer, offset, length, position)
```
Or:
```
fs.write(fd, buffer[, offset[, length[, position]]], callback)
fs.write(fd, string[, position[, encoding]], callback)
fs.writeFile(file, data[, options], callback)
fs.writeFileSync(file, data[, options])
fs.writeSync(fd, buffer[, offset[, length[, position]]])
fs.writeSync(fd, string[, position[, encoding]])
````
Should we make an exception now?

Reply:
This seems wrong to me, but... ok.